qutebrowser/browser/commands.py

```diff
@@ -832,8 +832,9 @@ def paste(self, sel=False, tab=False, bg=False, window=False):
             self._open(url, tab, bg, window)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
-                       count='count')
-    def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
+                       name=['tab-focus', 'b'], count='count',
```

Comment:
How important is the `b` alias to you? vim compatibility for commands never really was a concern, and a `b` command is rather unintuitive, so I think I'd prefer users just setting their own aliases if they want to (though there's a long-standing issue with completions for aliases...)